### DISCUSSION TITLE
time picker

### DISCUSSION THREAD

**Discussion by edlofau** [creativetimofficial/soft-ui-dashboard#35] [Q&A] [answered]
Hi, 
Is it any way to set the minutes step of the timepicker component to 15minutes instead 1?
Many thanks!

**Comment by groovemen** [comment, marked as answer]
Hello @edlofau, 

Thank you for using our products, we have checked the plugin's documentation and the step of the timepicker is 5 minutes: https://flatpickr.js.org/examples/#time-picker 
All the best,
Stefan

**Reply by edlofau** [reply]
Thanks for your reply Stefan, 

I see 1min increment in my form:
![image](https://user-images.githubusercontent.com/40147135/160902049-760e1b28-1f53-4a02-8dd1-a7b148aeb419.png)

I am using the example provided on https://www.creative-tim.com/learning-lab/bootstrap/forms/soft-ui-dashboard:
```
<div class="form-group">
        <label for="example-time-input" class="form-control-label">Time</label>
        <input class="form-control" type="time" value="10:30:00" id="example-time-input">
</div>
```
This is calling flatpickr actually? In that case, where should I change the parameter minuteincrement?
Thanks!
Ed

**Reply by edlofau** [reply]
Hi, I noticed I am using a simple html input of type time. Nothing to do with flatpickr. I will use the step="300" param since it does more or less what i want.
Thanks anyway
Ed
